vendor/github.com/hlandau/passlib/README.md

```diff
@@ -0,0 +1,95 @@
+passlib for go
```

Comment:
These discrepancies are due to the fact I didn't update the vendors globally but only added the necessary ones (i.e. the one mentioned in the PR comment)
Running:
```
make -f Makefile.refactor.make cleanall
make -f Makefile.refactor.make install-deps
```
updates the vendor folder globally, which is not my intent.